ydb/docs/en/core/maintenance/manual/maintenance-without-downtime.md

```diff
@@ -0,0 +1,98 @@
+    - No more than four unavailable VDisks, three of which must be in the same data center, are allowed for affected storage groups with the [mirror-3-dc](../../deploy/configuration/config.md#reliability) scheme. 
+    - No more than `(nto_select - 1) / 2` unavailable State Storage rings are allowed.
+- **Force** - forced mode, the failure model is ignored. Not recommended for use.
+
+### Priority {#priority}
+
+You can specify the priority of a maintenance task. A lower value means a higher priority.
+
+The actions of the task cannot be performed until all conflicting actions from tasks with a higher priority are completed. Tasks with the same priority have no advantage over each other.
+
+## Unavailable node limits {#unavailable-node-limits}
+
+In the CMS configuration, you can configure limits on the number of unavailable nodes for a database (tenant) or for the cluster as a whole. Relative and absolute limits are supported.
+
+By default, no more than 10% of unavailable nodes are allowed for each database and the cluster as a whole.
+
+## Checking algorithm {#checking-algorithm}
+
+To check if the actions of a maintenance task can be performed, the CMS sequentially goes through each action group in the task and checks the action from the group:
+- If the object of the action is a host, the CMS checks whether the action can be performed with all nodes running on the host. 
+- If the object of the action is a node, the CMS checks:
+    - Whether there is a lock on the node.
+    - Whether it's possible to lock the node according to the limits of unavailable nodes.
+    - Whether it's possible to lock all VDisks of the node according to the availability mode.
+    - Whether it's possible to lock the State Storage ring of the node according to the availability mode.
+    - Whether it's possible to lock the node according to the limit of unavailable nodes on which cluster system tablets can run.
+
+If the checks are successful, the action can be performed and a temporary locks are acquired on the checked nodes. The CMS then considers the next group of actions. Temporary locks help to understand whether the actions requested in different groups conflict with each other. Once the check is complete, the temporary locks are released.
+
+## Examples {#examples}
+
+The [ydbops](https://github.com/ydb-platform/ydbops) utility tool uses CMS for cluster maintenance without downtime. You can also use the CMS directly through the [gRPC API](https://github.com/ydb-platform/ydb/blob/main/ydb/public/api/grpc/draft/ydb_maintenance_v1.proto).
+
+### Take out a node for maintenance {#node-maintenance}
+
+{% note info "Functionality in development" %}
+
+Functionality is expected in upcoming versions of ydbops.
+
+{% endnote %}
+
+To take out a node for maintenance, you can use the command:
+```
+$ ydbops node maintenance --host <node_fqdn>
+```
+When executing this command, ydbops will acquire an exclusive lock on the node in CMS.
```

Comment:
This paragraph sounds weird: typically, multiple YDB nodes are on a single host, and such a command probably needs to acquire locks for all of them. Or maybe this implies only static/storage nodes, but this would need to be explicitly clarified, too.

Reply:
Changed parameter to `<node_id>` to make it more clear. I just took this command from https://github.com/ydb-platform/ydbops/issues/2
It is under active development. When the final variant of the command is chosen, the text in this article will be changed.

Reply:
@pixcc I don't think this change fixes the problem: in this article, we **shouldn't** use "node" and "host" interchangeably. This command sounds more like a maintenance of all (or some) nodes on a given host but not of a single node. Meanwhile, the surrounding text sounds like it is about a single node.

Reply:
I removed command to eliminate ambiguity.

It's not 100% clear whether the final version of the command will take out a host for maintenance (with all nodes) or individual nodes, so I've settled on a simple node case for now.